docs/install/README.md

```diff
@@ -44,9 +44,17 @@ Kata packages are provided by official distribution repositories for:
 
 ### Snap Installation
 
-The snap installation is available for all distributions which support `snapd`.
+> **WARNING:**
+>
+> The Snap package method is **unmaintained**. Use an
```

Comment:
I would add that it only delivers an old version of Kata, and specify which one.

Reply:
Good call @c3d - branch updated. Note the wording about the latest release. That might go stale by the time we raise a follow-up PR to remove the snap entirely, but I think it gets the point across that the snap is _old_ :smile:

Reply:
With that change, approved.